bots/openprescribing/openprescribing.py

```diff
@@ -29,10 +29,14 @@ def wrapper(message):
         if flags.deploy_suppressed:
             start_time, end_time = flags.deploy_suppressed
             if start_time <= now <= end_time:
-                message.reply(
-                    "Not deploying: suppressed until {}".format(
-                        end_time.strftime(TIME_FMT)
-                    ))
+                msg = ""
+                if not message.body['ts']:
```

Comment:
Could you add a comment here about why `'ts'` being in `message.body` means a PR has been merged?